modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -938,6 +938,29 @@ open class PlaybackManager @Inject constructor(
 
     fun toggleChapter(select: Boolean, chapter: Chapter) {
         launch {
+            getCurrentEpisode()?.let { episode ->
+                when (episode) {
+                    is PodcastEpisode -> {
+                        episodeManager.findByUuid(episode.uuid)?.let {
```

Comment:
If we have an `episode` from `getCurrentEpisode()` why do we need to fetch it again with `find()`? Are there any caching issues?

Reply:
I was trying a different approach earlier where this was needed. I removed it in https://github.com/Automattic/pocket-casts-android/pull/1856/commits/6138d59cc877e7f10596ef606a886910131585ae it doesn't seem to be needed now.